paper/paper.md

```diff
@@ -0,0 +1,188 @@
+
+OpenTURNS implements numerous reliability algorithms for rare-event
+probabilities: classic algorithms such as the First-Order Reliability
+Method, Second-Order Reliability Method, subset sampling, and other
+variance reduction methods, as well as more recent ones such as
+Nonparametric Adaptive Importance Sampling (NAIS) and Importance Sampling by Cross-Entropy.
+An event can be a single event (for example, threshold exceedance), a
+system event, or an event based on a stochastic process.
+
+For supervised learning, OpenTURNS enables the construction
+of surrogate models from data, such as linear regression models,
+polynomial chaos expansion, and Gaussian process regression.
+This surrogate model can also be built for models with functional inputs.
+The optimization of the hyperparameters of Gaussian process regression
+can be performed on an arbitrary subset of the hyperparameters.
+
+In many cases, probabilistic analyses use optimization algorithms (e.g.,
+for supervised learning or calibration).
+OpenTURNS can be used to define a wide variety of optimization problems
+in a unified manner and provides users with solution algorithms from
+numerous existing libraries: NLOpt, Ipopt, PAGMO, Bonmin, Dlib,
+Ceres, Cminpack, Cobyla, and more.
+It can solve optimization problems with single
+or multiple objectives, with equality or inequality constraints, and
+with continuous or discrete variables.
+
+Finally, since many analyses require the computation of integrals,
+OpenTURNS provides powerful integration methods based on sampling
+or quadrature methods: multivariate Gaussian quadrature (e.g.,
+Gauss–Legendre quadrature), Gauss–Kronrod, Smolyak, simplicial cubature,
+and other algorithms including those from the Cuba library.
+
+# Usages
+
+OpenTURNS is widely used in research, industrial engineering, and
+academic education.
+It has been integrated into university curricula (e.g., DTU Denmark) and
+engineering schools (e.g., ENSTA, ENPC, Centrale Nantes, and others)
+for courses on reliability analysis, risk assessment, and UQ.
+Stable releases are published on a bi-annual basis, and the library is
+distributed through Conda, pip, and Debian repositories and available in 
+Linux, Windows and MacOs.
+
+The library has been introduced in several publications (e.g.,
+[@baudin:2016]).
+Its calibration features, for example, are introduced in [@baudin:2021].
+The library is also used in several external packages including
+the benchmarking module otbenchmark [@fekhari:2021].
+Numerous application articles are based on OpenTURNS (e.g., [@Dutfoy:2023]
+on seismic risk assessment using the Gutenberg–Richter law).
+
+The Persalys graphical user interface [@persalys], built on top of OpenTURNS,
+enables users to perform UQ analyses with no (or minimal) Python
+programming.
+
+An annual Users’ Day has been held since 2008, fostering collaboration
+and knowledge exchange among researchers, engineers, and educators.
+
+# References
```

Comment:
We should add some references coming from Onera, Airbus and Phimeca....
@m-balesdent , @jschueller , @l-brevault , @sofianehaddad , @regislebrun : could you please add some publications and courses using OpenTURNS?
I think we can then submit this text.